.github/workflows/publish-api-reference.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish API References
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish-documentation:
+    permissions:
+      id-token: "write"
+      pages: "write"
+
+    environment:
+      name: "github-pages"
+      url: "${{ steps.deployment.outputs.page_url }}"
```

Comment:
is this used anywhere ?

Reply:
the url generated in deployment step is assigned to this `url`